starknet/src/proposal_validation_strategies/merkle_whitelist.cairo

```diff
@@ -0,0 +1,44 @@
+#[starknet::contract]
+mod MerkleWhitelistProposalValidationStrategy {
+    use sx::interfaces::IProposalValidationStrategy;
+    use serde::Serde;
+    use sx::types::UserAddress;
+    use array::{ArrayTrait, Span, SpanTrait};
+    use option::OptionTrait;
+    use sx::utils::merkle::{assert_valid_proof, Leaf};
+
+    const LEAF_SIZE: usize = 4; // Serde::<Leaf>::serialize().len()
+
+    #[storage]
+    struct Storage {}
+
+    #[derive(Drop, Serde)]
+    struct Input {
```

Comment:
could call something like `DeserializedParams`, which we could use in all proposal validation strategies so its clear what this is